app/src/test/java/ar/scacchipa/twittercloneapp/viewmodel/AuthWebDialogViewModelTest.kt

```diff
@@ -57,11 +62,8 @@ class AuthWebDialogViewModelTest {
         Assert.assertTrue(
             subject.responseDomain.value ==
                     ResponseDomain.Success(
-                        UserAccessTokenDomain(
-                            tokenType = "bearer",
-                            expiresIn = 7200,
+                        Credential(
```

Comment:
Should not be dealing with Credential class, credential is a class on the data layer, viewmodels should interact with clases that belongs to the domain layer.